### ISSUE TITLE
Splash screen background by theme

### ISSUE THREAD

**Issue by nvllz** [yamin8000/freeDictionaryApp#78]
Hi, I just got flashed when wanted to search for some term clarification within your great app. For better experience, the splash screen background color should match the current theme setting.

**Comment by yamin8000** [comment]
> Hi, I just got flashed when wanted to search for some term clarification within your great app. For better experience, the splash screen background color should match the current theme setting.

Hi.
Thank you.
I just want to clarify something, did you experience a white flash during launch/splash or after the app is opened and you're searching for a new term?

**Comment by yamin8000** [comment]
> It's the splash screen (background when opening the app) which is always white regardless of the current theme. When using dark theme it would be good to have it black to prevent such cases.

Hmm. That never happens to me. Which version of Android you are using?

**Comment by yamin8000** [comment]
> Android 13, you can learn more here: https://developer.android.com/develop/ui/views/launch/splash-screen

Thanks. I'll look into it.

**Comment by yamin8000** [comment]
> Android 13, you can learn more here: https://developer.android.com/develop/ui/views/launch/splash-screen

TL;DR;  
Hell, I added it anyway.

One big problem with that splash screen is its complete disregard for the Jetpack Compose Theme System. This type of Splash Screen still relies on XML configuration while Jetpack Compose apps' colors are defined inside Kotlin codes so its config is always prone to be out of sync with app theme colors also it's not aware of the app's current theme so its understanding of whether an app is in dark mode or light mode is based on device dark/light setting.
